pkg/picker/config.go

```diff
@@ -7,13 +7,14 @@ import (
 
 // Config is the configuration to build Picker
 // The priority is as follows:
-//		ConcatItems > Indices
-// 		Nullable
-// 		DefaultValue
-//    NullValue, if set to null, subsequent conversions will be skipped.
-// 		Type
-// 		Function
-// 		CheckOnPost
+//
+//			ConcatItems > Indices
+//			Nullable
+//			DefaultValue
+//	   NullValue, if set to null, subsequent conversions will be skipped.
+//			Type
+//			Function
+//			CheckOnPost
```

Comment:
Please revert the changes here, thank you!

Reply:
sorry, I don't know why my format tool did this. I have fixed it.